tools/validators/instance_validator/validate/entity_instance.py

```diff
@@ -168,45 +168,38 @@ def _ValidateTranslation(self):
         print('Invalid extra field present:', field_name)
         return False
 
+      valid_units = self.universe.GetUnitsMapByMeasurement(field_name.data)
       translation_map = translation_body[field_name]
-      valid_units = self.universe.unit_universe.GetUnitsMap('').keys()
       valid_states = self.universe.state_universe.GetStatesMap('').keys()
 
       # check if keys are UDMI compliant then skip the units and states
       if isinstance(translation_body[field_name].data, str):
         continue
 
-      # three remaining possibilities for translation format
-      if self.unit_values in translation_map.keys():
-        unit_values_map = translation_map[self.unit_values]
-        for unit in unit_values_map.keys():
-          if unit not in valid_units:
-            print('Invalid translation unit', unit)
-            return False
-      elif self.states in translation_map.keys():
-        states_map = translation_map[self.states]
-        for state in states_map.keys():
-          if state not in valid_states:
-            print('Invalid translation state', state)
-            return False
-      elif self.units in translation_map.keys():
-        # check that the unit map has keys named `keys`, `values`
-        units_map = translation_map[self.units]
-        if self.key not in units_map.keys():
-          print('Invalid units translation is missing key', self.key)
-          return False
-        if self.values not in units_map.keys():
-          print('Invalid units translation is missing values', self.values)
-          return False
-
-        unit_values_map = units_map[self.values]
-        for unit in unit_values_map.keys():
-          if unit not in valid_units:
-            print('Invalid translation unit', unit)
-            return False
-      else:
-        print('Translation has improper keys:', field_name)
-        return False
+      if valid_units:
```

Comment:
note: this needs to be more specific to differentiate between no-units-universe and this field does not have any valid units (i.e. empty vs. none)

Reply:
if there is no units universe can someone put units in the config file?